sdk/notificationhubs/Azure.ResourceManager.NotificationHubs/src/Generated/NotificationHubNamespaceData.cs

```diff
@@ -13,16 +13,16 @@
 
 namespace Azure.ResourceManager.NotificationHubs
 {
-    /// <summary> A class representing the NamespaceResource data model. </summary>
-    public partial class NamespaceResourceData : TrackedResourceData
+    /// <summary> A class representing the NotificationHubNamespace data model. </summary>
+    public partial class NotificationHubNamespaceData : TrackedResourceData
```

Comment:
The property name `NamePropertiesName` look weird, I think we can rename it.

Reply:
fixed